engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/VolumeOrchestrator.java

```diff
@@ -1346,13 +1346,14 @@ private void checkConcurrentJobsPerDatastoreThreshhold(final StoragePool destPoo
 
     @Override
     @DB
-    public Volume migrateVolume(Volume volume, StoragePool destPool) throws StorageUnavailableException {
+    public Volume migrateVolume(Volume volume, StoragePool destPool, DiskOffering newDiskOffering) throws StorageUnavailableException {
```

Comment:
@SadiJr Is this new offering updated on the volume record, when the migrate is successful.

Reply:
@sureshanaparti, I did not get what you mean by this, we set the payload with this disk offering, for setting the IOPS later.